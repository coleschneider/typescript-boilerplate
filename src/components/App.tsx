import * as React from 'react';

class App extends React.Component {
  render() {
    return <div className="wrapper">Hello !</div>;
  }
}

export default App;
