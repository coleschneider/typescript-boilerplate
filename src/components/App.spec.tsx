import * as React from 'react';
import App from './App';
import { shallow } from 'enzyme';

describe('App', () => {
  it('Should render', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
