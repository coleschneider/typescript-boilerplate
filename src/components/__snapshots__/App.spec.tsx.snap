// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Should render 1`] = `
<BrowserRouter>
  <div>
    <Component />
    <Header />
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        render={[Function]}
      />
    </Switch>
  </div>
</BrowserRouter>
`;
